test/extern/kbrady/extern_array_call.chpl

```diff
@@ -1,3 +1,4 @@
+use CPtr;
```

Comment:
Just a note to myself that this is a case that could be removed once #16452 is incorporated.